hubitat-mitsubishi-mqtt.groovy

```diff
@@ -1,7 +1,7 @@
 /**
  * Hubitat Device Driver
  * Mitsubishi Heat Pump + MQTT
- * v1.1.3
+ * v1.1.4
```

Comment:
version number should usually be bumped in its own commit after we roll up any changes :)